tests/mail/tests.py

```diff
@@ -393,6 +426,28 @@ def test_to_in_headers_only(self):
         ).message()
         self.assertEqual(message["To"], "to@example.com")
 
+    def test_to_not_repeated(self):
+        """
+        Make sure that to is not repeated (#35033)
+        """
```

Comment:
Please remove ticket numbers from docstrings. Also, to add more explanation, docstrings should state the expected behavior and omit prefixes like "Make sure" since all tests ensure things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Personally, I'd remove docstring from new test methods as test method names are self-explanatory.